### ISSUE TITLE
Go Project Builder error since upgrade to 0.12.0

### ISSUE THREAD

**Issue by dmuntean** [GoClipse/goclipse#147]
I'm getting following error on each project builder invocation:

```
Errors occurred during the build.
Errors running builder 'Go Project Builder' on project 'ppp-test'.
AssertFailedException
```

And that happens on every save and all go projects I have in my workspace. I tried creating a brand new go project, but it still fails with the same error.

The following is eclipse error log:
```
!ENTRY org.eclipse.core.resources 4 75 2015-08-06 16:23:34.404
!MESSAGE Errors occurred during the build.
!SUBENTRY 1 com.googlecode.goclipse.core 4 75 2015-08-06 16:23:34.404
!MESSAGE Errors running builder 'Go Project Builder' on project 'ppp-test'.
!STACK 0
AssertFailedException
        at melnorme.utilbox.core.Assert$AssertHandler.handleAssert(Assert.java:60)
        at melnorme.utilbox.core.Assert.checkAssertion(Assert.java:87)
        at melnorme.utilbox.core.Assert$AssertNamespace.assertTrue(Assert.java:149)
        at melnorme.lang.ide.core.operations.LangProjectBuilder.prepareForBuild(LangProjectBuilder.java:99)
        at melnorme.lang.ide.core.operations.LangProjectBuilder.build(LangProjectBuilder.java:120)
        at org.eclipse.core.internal.events.BuildManager$2.run(BuildManager.java:734)
        at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
        at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:205)
        at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:245)
        at org.eclipse.core.internal.events.BuildManager$1.run(BuildManager.java:300)
        at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
        at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:303)
        at org.eclipse.core.internal.events.BuildManager.basicBuildLoop(BuildManager.java:359)
        at org.eclipse.core.internal.events.BuildManager.build(BuildManager.java:382)
        at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:144)
        at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:235)
        at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
```

**Comment by bruno-medeiros** [comment]
Hum, I can replicate it, but its not a trivial fix. In the meanwhile, a workaround is to disable Eclipse's auto-build on save (menu: `Project / Build Automatically`).

**Comment by bruno-medeiros** [comment]
Fixed on a 0.12.0 re-release.
